windows.md

```diff
@@ -397,7 +397,64 @@ With those compatibility things out of the way, you're ready to start the system
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
+
+    if opening Docker Desktop or running the command above shows a message that says `Docker Desktop - Windows HyperVisor is not Present`, you may need to enable virtualization on your machine.
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    First check if virtualization is enabled on your computer on the **Performance** tab of the Task Manager (open the Task Manager with <kbd>Ctrl</kbd>-<kbd>Shift</kbd>-<kbd>Escape</kbd>):<br><br>
+
+    <img src="windows-7-virtualisation-enabled.avif"><br><br>
+
+    Continue with the steps below if virtualization is not enabled, Enable virtualization in your BIOS or UEFI:
+
+    1. Restart your computer if it is already started or start it, if not
+    2. Right when the computer is starting up (Still on the black screen befor the computer brand logo blashes), press repeatedly any of these keys: <kbd>Delete</kbd>, <kbd>Esc</kbd>, <kbd>F1</kbd>, <kbd>F2</kbd>, <kbd>F4</kbd>, <kbd>F10</kbd>, or <kbd>F12</kbd>. Each computer manufacturer uses a different key but it may show a brief message at boot time telling you which one to press. If you miss it the first time, restart and try again. If you are not able to enter the BIOS via this method, check this [BIOS key lists](https://www.disk-image.com/faq-bootmenu.htm) for different computers or consult your computers manual
```

Comment:
`Still` lowercase, and `before` typo

Reply:
> For me, when I see the brand logo of my motherboard, I can still press the keys to open BIOS

I need to start pressing the BIOS button in the two computers I used before the logo shows; if not, the computer will have a normal boot. If you can start the BIOS mode by pressing the key before the logo shows, I will leave it like that, as that would be when to start initializing BIOS in most computers

`blashes` meant `flashes`
ab0172064419fd1490faa93d3f593f58dfe08d41

48b4e634c0800e7304f89cc2d21482266540de1b

Reply:
I created a small video and added it to my [comment](https://github.com/upleveled/system-setup/pull/16#pullrequestreview-1988781209)

Reply:
Yes, thank you so much. Your computer shows brief instructions on what key to press when booting. I think this is covered in the statement I made here

> Each computer manufacturer uses a different key, but it may show a brief message at boot time telling you which one to press